themes/index.js

```diff
@@ -416,6 +416,13 @@ export const themes = {
     border_color: "85A4C0",
     bg_color: "030314",
   },
+  github_dark_dimmed: {
```

Comment:
Maybe let's place this under the `github_dark` theme?

https://github.com/anuraghazra/github-readme-stats/blob/542a2326ad2cfe03a7bc36d5cb75dc86c4da3b70/themes/index.js#L318-L323

Reply:
I put it at the end of the file because of the contribution guidelines. But I agree with the suggestion. ;)